perun-base/src/main/java/cz/metacentrum/perun/core/api/RoleManagementRules.java

```diff
@@ -34,6 +34,8 @@
  * skipMFA is a flag that whether the role should skip MFA check.
  * mfaCriticalRole is a flag marking roles always requiring MFA from users having that role
  * displayName is a more user-friendly name
+ * receiveNotifications contains names of objects for which the role should get notifications
+ * 			   Example value: VO ; meaning: will receive notifications when vo application is created/failed
```

Comment:
I think it would be nice to also add this to the perun-roles and https://github.com/CESNET/ansible-role-perun-docker-server

Reply:
thanks, added. also missed the docs in perun-roles.yml, added now.